### PR TITLE
Remove readmeio

### PR DESCRIPTION
**Pull request by JamesDrummond**
Related to https://github.com/codenvy/codenvy/issues/1785

This refers to external documentation at https://codenvy.com/docs/. We can enhance this PR with internal documentation which I do not know how to reference(relative/full path unknown) or we can merge this and enhance later.

### CODE REVIEW

**Comment by slemeur** [comment]
@ashumilova + @JamesDrummond instead of having harcoded links, should we redirect to docs embedded in the product?

**Comment by JamesDrummond** [comment]
@slemeur That was my thought too but didn't know how. I would say let's merge this then create new issue for internal links if it takes too long to implement.

File: dashboard/src/app/factories/factory-details/information-tab/factory-information/factory-information.html [hunk 1]
```diff
@@ -177,7 +177,7 @@
       </md-content>
       <div layout="row" flex>
         <div>
-          <a href="https://codenvy.readme.io/docs/factories" target="_blank">Factory configuration docs</a>
+          <a href="https://codenvy.com/docs/integration-guide/workspace-automation/index.html" target="_blank">Factory configuration docs</a>
```
Comment (slemeur) [inline dashboard/src/app/factories/factory-details/information-tab/factory-information/factory-information.html]: should be 
href="/docs/integration-guide/workspace-automation/index.html"
to point to the docs in the product

**Comment by slemeur** [comment]
Ok I just posted a comment with how it should look.
Proceed as on https://github.com/eclipse/che/pull/4166

**Comment by JamesDrummond** [comment]
@slemeur Please take a look one last time.

**Pull request merged**